app/views/static_pages/home.html.erb

```diff
@@ -1,7 +1,8 @@
-<% provide(:title, "Home") %>
 <h1>Sample App</h1>
 <p>
     This is the home page for the
     <a href="https://www.railstutorial.org/">Ruby on Rails Tutorial</a>
     sample application.
-</p>
\ No newline at end of file
+</p>
+
+
```

Comment:
bỏ dòng 1 dòng thừa này nhé e, thường có rule là end of file thì có 1 empty line, chỉ 1 thôi